spherical_geometry/polygon.py

```diff
@@ -938,22 +941,22 @@ def from_cone(cls, lon, lat, radius, degrees=True, steps=16):
         return cls((polygon,))
 
     @classmethod
-    def from_wcs(cls, fitspath, steps=1, crval=None):
-        r"""
-        Create a new `SphericalPolygon` from the footprint of a FITS
-        WCS specification.
+    def from_wcs(
+        cls, wcs: gwcs.WCS | astropy.wcs.WCS | astropy.io.fits.Header | str, edges_per_side: int = 1, crval=None
```

Comment:
Should `edges_per_side` be added at the end to avoid immediate API breakage since this API allows people to pass in keyword as arguments?

```suggestion
        cls, wcs: gwcs.WCS | astropy.wcs.WCS | astropy.io.fits.Header | str, crval=None, edges_per_side: int = 1
```